projects/pillow/fuzz_pillow.py

```diff
@@ -0,0 +1,47 @@
+
+def TestOneInput(data):
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+    # We need to write a jpeg file first because otherwise libFuzzer will complain that no coverage eixsts
+    # We must trigger execution of the native code even in the first empty try-out runs.
+    if os.path.isfile("/tmp/newfile"):
+        os.remove("/tmp/newfile")
+    with open("/tmp/newfile", "wb+") as rb:
+        rb.write(encoded3[0:1000])
+
+    # Now append our random data
+    with open("/tmp/newfile", "ba+") as rb:
+        rb.write(data)
+
+    # Fuzzer entrypoint
+    with Image.open("/tmp/newfile", "r") as f:
```

Comment:
Please catch `Exceptions`, since they are expected ( Pillow's exceptions are directly inheriting from `Exceptions`, so there is no way to catch them all without enumerating them).

Reply:
Thanks!